docs/notebooks/mango-python-coverage.ipynb

```diff
@@ -0,0 +1 @@
+../../example-files/notebooks/mango-python-coverage.ipynb
```

Comment:
Just curious, how is this working? Is it executing the notebook somewhere?

Reply:
Yes, when you run
`make html` in the `doc/` directory
nbsphinx is running `nbconvert` and executes the notebook, then converting it into html, which is then part of the sphinx generated site.

This works great from the matplotlib notebooks, but still no "html" static version of the mango-viz browser widget is generated, despite that the notebook itself will be run if included.  Matplotlib images end up in docs/_build/html/images after sphinx runs.  Probably we would need to add some function to the widget object to know to draw itself as a png....  
We could still manually make a pre-executed mango-viz notebook with an image embedded from a screenshot ourselves...

Reply:
Is nbsphinx running Spark then to execute the notebook?

Yes, I think for the mango-viz we can just make some documentation by hand, and not do nbshinx.

Reply:
Yes, Spark is being run.
I will make the pre-made mango-viz note book with a static screenshot-ed click of the widget.

Reply:
awesome thanks @jpdna!

Reply:
It looks like Spark is not being run @jpdna . `make html` generates the error:

```
Notebook error:
CellExecutionError in notebooks/mango-python-alignment.ipynb:
------------------
try:
    spark
except NameError:
    from pyspark.sql import SparkSession
    spark = SparkSession.builder.master("local").getOrCreate()
try:
   ac
except NameError:
   ac = ADAMContext(spark)

# load file
alignment = ac.loadAlignments(alignmentFile)
------------------

TypeError: 'JavaPackage' object is not callable
```

Meaning it cannot find the required jars. For readthedocs to be able to execute this, they would first have to build the entire project, and then run the notebooks with a SparkContext. I would recommend [testing this in Docker ](https://hub.docker.com/r/readthedocs/build/) for a more accurate idea of how this will work on readthedocs servers.